### DISCUSSION TITLE
Slide Content Displayed Too Small After Upgrade

### DISCUSSION THREAD

**Discussion by ryanlo713** [vuepress-theme-hope/vuepress-theme-hope#4334] [Q&A] [answered]
**Description:**

After upgrading to the latest version, I've noticed that the slide content is displayed at a much smaller scale than expected.

**Comparison of Version Numbers:**
![Comparison of Version Numbers](https://github.com/user-attachments/assets/028ec8f1-0947-4af3-9a06-dd6d32702890)

**Issue Detail:**
As demonstrated in the attached screenshot, the content is scaled too small, making it difficult to read and follow along with the presentation. This issue also occurs in the slide examples of the official documentation, which indicates that it may be a broader problem.

**Attachments:**
![screenshot showing the scaling issue](https://github.com/user-attachments/assets/f7c072c5-36da-425c-a8f1-ab0b699e606f)
![another screenshot showing the same problem in the official documentation slides](https://github.com/user-attachments/assets/d63368d1-b726-4ccb-922c-0bf5ecabca80)

Please investigate this issue, as it severely impacts readability.

Thank you!

**Comment by Mister-Hope** [comment, marked as answer]
Can you open an issue for this?  In case might I forgot it.

Also just a reminder,  I was busy these days and you can see that there is new version coming in the past two weeks at least.  So I can't guarantee when I had time to fix this if you have time to investigate, pr is GREATLY welcomed.
